cardinal/message/manager.go

```diff
@@ -1,27 +1,31 @@
 package message
 
 import (
+	"reflect"
+	"slices"
+
 	"github.com/rotisserie/eris"
 	"pkg.world.dev/world-engine/cardinal/types"
-	"slices"
 )
 
 type Manager struct {
-	registeredMessages map[string]types.Message
-	nextMessageID      types.MessageID
+	registeredMessagesByName map[string]types.Message
```

Comment:
with the removal of `registeredMessagesByType` we can change this back to `registeredMessages` and keeping it consistent with the rest of the managers

Reply:
sure don't mind.